nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors/pom.xml

```diff
@@ -107,5 +107,23 @@
             <version>1.10.0-SNAPSHOT</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
I would add groovy-dateutil, groovy-datetime, groovy-templates, groovy-sql and now I think there is groovy-json-direct as well. Also, that needs to be applied to the groovyx bundle.

Reply:
Groovy changes were pulled out into a separate [PR 3547](https://github.com/apache/nifi/pull/3547).